elisp/Makefile

```diff
@@ -5,6 +5,10 @@
 ### Created: Nov 18, 1994
 ### Revised: Feb 10, 2023
 
+EMACS = emacs
+prefix = /usr/local
+elispdir = $(prefix)/share/emacs/site-lisp/mew
```

Comment:
Why do you use elispdir instead of ELISPDIR?

Reply:
I don't have a strong reason, just see the install-elisp target,
not familiar with the conventions.